src/mmw/apps/modeling/models.py

```diff
@@ -4,12 +4,16 @@
 from __future__ import division
 
 from django.db.models import FileField
+from django.conf import settings
 from django.contrib.gis.db import models
 from django.contrib.auth.models import User
 
 from apps.core.models import Job
 
 
+DRB = settings.DRB_PERIMETER.buffer(0.1)
```

Comment:
I compared "Headwaters Pequea Creek", which is outside the DRB:

![image](https://user-images.githubusercontent.com/1430060/84192168-99fbf080-aa67-11ea-99a7-fd2218eb710d.png)

and the next-door "Upper West Branch Brandywine Creek", which is inside the DRB:

![image](https://user-images.githubusercontent.com/1430060/84192251-b7c95580-aa67-11ea-9fa7-7c94f7958bf3.png)

to get this number. I don't know if I should go lower.